container_pipeline/scanners/pipeline.py

```diff
@@ -198,7 +198,7 @@ def process_output(self, json_data):
         data["scanner_name"] = self.scanner_name
         if json_data["Scan Results"]["Package Updates"]:
             data["logs"] = json_data
-            data["msg"] = "Container image requires update."
+            data["msg"] = "RPM updates available for the image."
```

Comment:
We are updating the image if we find some updated RPMs for containers. is my understanding correct to say this is redundant to the RPM update feature?

Reply:
Yes, I've been thinking the same since a few days. But there are a few bugs with RPM tracking. Once those are fixed, I plan to deprecate this scanner altogether.